docs/docs/install/kubernetes.md

```diff
@@ -47,17 +49,35 @@ tracee-fcjmp   1/1     Running   0          4m11s
 ```
 </details>
 
-## Interacting with Tracee
+Once installed, Tracee immediately starts producing events. Since Tracee is deployed as a DaemonSet, a Tracee Pod is running on every node in the cluster. Every Tracee Pod is monitoring the node it is running on.
 
-Once installed, Tracee immediately starts producing system activity events, such as processes and containers activity, network activity, and more. To see the events that Tracee produces, use can use the `kubectl logs` command.
+## Viewing Events
+
+The easiest way to tap into the log stream of all Tracee Pods is with the `kubectl logs` command:
 
 ```console
-kubectl logs -f daemonset/tracee -n tracee-system
+kubectl logs -f daemonset/tracee
 ```
 
-In production scenario you would want to collect and ship the events to a persistent storage.
+!!! Note
+    Tracee can produce a very high volume of events which could overwhelm kubectl's log collection command. If run in a busy cluster or with a verbose policy, this command might be slow or unresponsive.
+
+In production scenario you would probably want to collect and ship events logs into a persistent storage that you can query.   
+You can use any log collection solution of your choosing. We have a tutorial on how to do this using the open source Graphana Stack [here](../../tutorials/deploy-grafana-dashboard.md).
+
+## Applying Policies
+
+By default, Tracee collects a basic set of events that gives you a general overview of the cluster. If you're looking to do more with Tracee, You might want to create a new [Policy](../policies/index.md). A policy lets you capture specific set of events from a specific set of workloads. For example, if you have an application that you want to monitor more closely, or in a specialized way, you can create a policy scoped to that application, with a different set of events and filters applied. To learn more, please refer to the [Events](../events/index.md) and [Policies](../policies/index.md) sections.
+
+When you are ready to apply a policy, it's as easy as `kubectl apply -f your-policy.yaml`. More details [here](../policies/usage/kubernetes.md).
 
-## Exercising a security event
+## Configuring Tracee
+
+In some cases you will need to configure Tracee to your preferences. For example, to change the output event format, or to set a different log level. To learn more about available configuration options please see the [configuration](../install/config/index.md) section.
+
+When you are ready to make a configuration change, it's as easy as `kubectl edit cm -n tracee tracee`. More details [here](../install/config/kubernetes.md).
```

Comment:
As we show how to install with helm, I would prefer said one can customize the config through helm. Something like:

```
You can inject your own config file if installing with helm, with `helm .. --set-file traceeConfig=/path/to/file`, or if you want just to customize one option, our helm supports each config option individually, for example to increase your cache size `helm install .. --set config.cache.size=1024`
```

Reply:
I took your suggestion to use helm but didn't explain all the possible options (helm values file, helm value override, kustomize), since I think these more complete information belongs in the `install/config/kubernetes` doc